content/advanced/330_servicemesh_using_appmesh/integrate_observability/cloudwatch_logs.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Cloudwatch Logs"
+date: 2020-01-27T08:30:11-07:00
+weight: 92
+draft: false
+---
+
+#### Cloudwatch Console
+
+Logs are collected by the fluentd daemonset running in the EKS nodes. The following CloudWatch log groups are created by default when Container Insights is setup:
```

Comment:
I know you get to it below, but maybe good to call out that fluentbit is also running to collect logs and ship them to CW for Fargate.

Reply:
updated with details